### PR TITLE
Fix for issue - Responsive Behaviour not being shown in Toggle Buttons

### PR DESCRIPTION
**Pull request by puneetpalkaur**
fixed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@
   <h1 class="text-center page-title">APJ Abdul Kalam</h1>
     <h2 class="text-center sub-title"><i>The Missile Man of India</i></h2>
 
-   <div class="row slideshow text-center">
+   <div class="row slideshow text-center" style="align-items: center;display: inline-flex;">
```
Comment (jaykay12) [inline index.html]: screenshot from 2018-10-30 23-09-26

The entire row containing image and toggles is losing the central allignment. 
This is not something which we intend to handle separately, kindly central align the row and push commits to this PR only.

**Review by jaykay12 — CHANGES_REQUESTED** [review]
Add changes as suggested, the PR is good to be merged then after.

**Comment by puneetpalkaur** [comment]
center alignment fixed

File: index.html (hunk 1 above)
Comment (puneetpalkaur) [reply index.html]: gotcha! its fixed , lmk if anythin else needs to be changed

**Pull request closed without merging**